app/components/candidate_interface/covid19_dashboard_banner_component.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="app-banner govuk-!-margin-bottom-7">
+  <div class="app-banner__message">
+    <h2 class="govuk-heading-m">Coronavirus (COVID-19)</h2>
+    <p class="govuk-body">
+      Training providers may take longer to respond to your application.
+      You may also have longer to respond to any offers. You will be able to see your deadlines below.
+    </p>
+  </div>
```

Comment:
```suggestion
    <p class="govuk-body">
      Training providers will take longer to respond to your application.
      You will also have longer to respond to any offers. You will be able to see your deadlines below.
    </p>
  </div>
```

Not sure who authored this content. cc @EmmaFrith

Reply:
Suggestion to make the content more assertive.

Reply:
I'll check in with Emma F.

Reply:
can't really say will because not everyone will have extended deadlines

Reply:
can't really say will because not everyone will have extended deadlines

Reply:
I thought it was best practice not to indicate layout in messages, because for screen reader users and others, words like ‘below’ don’t mean a great deal?

Reply:
Ah, should we change this back to "here" then?

Reply:
i would just say 'on this page'

Reply:
https://ukgovernmentdfe.slack.com/archives/CN9Q1VB2M/p1585042323027100